src/componentSpec.ts

```diff
@@ -199,6 +199,19 @@ export interface TaskOutputArgument {
   };
 }
 export type ArgumentType = string | GraphInputArgument | TaskOutputArgument;
+
+/**
+ * Argument input to the Argument Editor.
+ */
+export type ArgumentInput = {
```

Comment:
Please, let's move this type out of this file. This `componentSpec.ts` file is unrelated to UX. Let's not mix the UX types and core business logic types.